src/include/hip/rpp_hip_common.hpp

```diff
@@ -1778,6 +1778,21 @@ __device__ __forceinline__ float rpp_hip_math_sinc(float x)
     return (fabsf(x) < 1e-5f) ? (1.0f - x * x * ONE_OVER_6) : sinf(x) / x;
 }
 
+__device__ __forceinline__ void rpp_hip_log(d_float8 *src_f8, d_float8 *dst_f8)
+{
+    for(int i = 0; i < 8; i++)
+        src_f8->f1[i] = (!src_f8->f1[i]) ? std::nextafter(0.0f, 1.0f) : fabsf(src_f8->f1[i]);
```

Comment:
Please add a comment as to why all this is added. 
Please ensure we add in a note in the header that states the domain and range of natural log and that the function does an abs by default. 
Also, has this been confirmed with any other open source lib?

Reply:
Added note in doxygen, usually they return NAN if input is 0. Reference software uses absolute to avoid negative inputs.